src/components/homeLeftArea/homeLeftArea.tsx

```diff
@@ -13,7 +13,7 @@ import { getProjectsSelector } from "@/redux/selector";
 const HomeLeftArea: React.FunctionComponent = () => {
   const dispatch = useDispatch();
   const projectListData = useSelector(getProjectsSelector);
-  const projectArray = projectListData[0]?.projects;
+  const projectArray = projectListData[0] ?.projects;
```

Comment:
Why there is a space here between projectListData[0] and  ?.projects

Reply:
while doing prettier code format the space is coming automatically